src/Subscriptions/SubscriptionBroadcaster.php

```diff
@@ -93,6 +93,7 @@ function (Subscriber $subscriber) use ($root): void {
                     $subscriber->args,
                     $subscriber->setRoot($root)
                 );
+                dump($data);
```

Comment:
do we need this here?

Reply:
This PR is a work of progress and currently not in a working state. So no the dump is not needed in the final PR, but while in this state it's fine, it's a draft PR after all.